docker/entrypoint.sh

```diff
@@ -59,6 +59,14 @@ if [ "$1" = "--build-engines" ]; then
   else
     echo "Engine for DepthAnything2 already exists, skipping..."
   fi
+
+  # Build Engine for Depth Anything2 (large)
```

Comment:
@pschroedl I've added the new large model as an additional engine to avoid disrupting the current infrastructure. We can remove the old base model later if we decide to fully transition. In our tests, FPS wasn't impacted, but we'll need to verify that on staging.

Reply:
We will need to add this new model to `dl_checkpoints.sh` here https://github.com/livepeer/ai-runner/blob/21c5539a014f32c7371892b9ae53bf81fb77821e/runner/dl_checkpoints.sh#L123-L132. As a side note, ai-runner should be able to make use of our setup_models script